.github/workflows/jules.yml

```
@@ -0,0 +1,80 @@
+      - name: Queue Task
+        if: ${{ steps.quota.outputs.usage >= steps.quota.outputs.quota }}
+        run: |
+          task='{ "subscription": "Jules_Agent", "issue_number": "${{ github.event.issue.number }}", "pr_url": "${{ github.event.pull_request.html_url }}" }'
+          python scripts/quota_manager.py add_task "$task"
+          gh issue comment ${{ github.event.issue.number }} --body "The daily task limit for Jules has been reached. Your request has been added to the queue and will be processed as soon as possible."
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Commit state changes
+        if: always()
+        run: ./scripts/commit_changes.sh "chore: update state for Jules task"
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)